app.py

```diff
@@ -237,18 +237,28 @@ def get_bundles(entry):
     root_dir = os.path.dirname(os.getcwd())
     build_dir = os.path.join("static", "build")
     asset_path = "/static/build/"
-    bundles = {"css": [], "js": []}
+    bundles = {"js": []}
     manifest_path = os.path.join(build_dir, "assets.json")
     with open(manifest_path) as manifest:
         assets = json.load(manifest)
     entrypoint = assets["entrypoints"][entry]
     for bundle in entrypoint["js"]:
         bundles["js"].append(asset_path + bundle)
-    for bundle in entrypoint["css"]:
-        bundles["css"].append(asset_path + bundle)
     return bundles
 
 
+def get_styles(entry):
+    root_dir = os.path.dirname(os.getcwd())
+    build_dir = os.path.join("static", "build")
+    asset_path = "/static/build/"
+    manifest_path = os.path.join(build_dir, "styles.json")
+    with open(manifest_path) as manifest:
+        assets = json.load(manifest)
+    css = assets[entry]
+    css_path = asset_path + css
+    return css_path
+
```

Comment:
@mailbackwards, I requested your review for all the python edits I made. Does this all check out on your end?
Generally speaking, I'm taking some JSON like this:
```
{
  "business": "business.1557767697331.css",
  "charge": "charge.1557767697562.css",
  "circle": "circle.1557767697751.css",
  "donate": "donate.1557767697982.css",
  "old": "old.1557767698239.css"
}
```
and passing that context to each route. Like on [this line](https://github.com/texastribune/donations/pull/192/files#diff-3f41e546893dc64b71aaacad12cad815R407) with `css = get_styles("donate")`

Reply:
This LGTM-- it could be a little bit brittle, for instance if someone puts a typo in one of the arguments then it will error out the whole page, if you'd rather it error more "silently" and show the page w/o CSS, you might want to make it more forgiving by checking for the key in the dict first, and/or type-checking before concatenating them. But if we did that we'd also want to manually throw an error to Sentry so that we'd know something was up, so it just adds lotsa complexity for a very unlikely case...so I think it's fine for it to error loudly.

Conceptually/longer-term, I could see drawing out `get_bundles` and `get_styles` into a common underlying function since they have plenty of overlap in usage+details, but probably overkill and outta scope here. WFM!

Reply:
@mailbackwards thank you and yes that's true on the "entry" param being a bit brittle. I was starting to feel like separating `get_bundles` and `get_styles` felt kinda repetitive. I'll see if I can add the `get_styles` extraction to `get_bundles`. Then when we fix the how that errors, we do it in one place. Good feedback, thanks!